src/main/java/de/tum/in/www1/artemis/service/QuizExerciseImportService.java

```diff
@@ -81,15 +81,15 @@ private QuizExercise copyQuizExerciseBasis(QuizExercise importedExercise) {
     }
 
     /**
-     * This helper method copies all questions of the {@code importedExercise} into a new exercise.
+     * This helper method copies all questions of the {@code sourceExercise} into a new exercise.
      *
-     * @param importedExercise The exercise from which to copy the questions
-     * @param newExercise      The exercise to which the questions are copied
+     * @param sourceExercise The exercise from which to copy the questions
+     * @param newExercise    The exercise to which the questions are copied
      */
-    private void copyQuizQuestions(QuizExercise importedExercise, QuizExercise newExercise) {
+    private void copyQuizQuestions(QuizExercise sourceExercise, QuizExercise newExercise) {
         log.debug("Copying the QuizQuestions to new QuizExercise: {}", newExercise);
 
-        for (QuizQuestion quizQuestion : importedExercise.getQuizQuestions()) {
+        for (QuizQuestion quizQuestion : sourceExercise.getQuizQuestions()) {
             quizQuestion.setId(null);
             quizQuestion.setQuizQuestionStatistic(null);
             if (quizQuestion instanceof MultipleChoiceQuestion mcQuestion) {
```

Comment:
The mutation is not relevant since it does not happen in a `@Transactional context`. We only remove unneeded data before copying, the source exercise is not changed.